gnc/navigator_controller/nodes/mrac_controller.py

```diff
@@ -96,11 +96,15 @@ def __init__(self):
         self.lin_vel = np.zeros(2)
         self.ang_vel = 0
         self.state = Odometry()
+
+        self.use_lqrrt = True
+
         # Subscribers
-        if self.use_external_tgen:
-            rospy.Subscriber("/trajectory", PoseTwistStamped, self.set_traj)
+        if self.use_lqrrt:
+            rospy.Subscriber("/lqrrt/ref", Odometry, self.set_traj_from_odom_msg)
         else:
-            rospy.Subscriber("/waypoint", PoseStamped, self.set_waypoint)
+            rospy.Subscriber("/trajectory", PoseTwistStamped, self.set_traj)
+
         rospy.Subscriber("/odom", Odometry, self.get_command)
         rospy.Subscriber('/learn', Bool, self.toggle_learning)
```

Comment:
True, that way everything that needs to change for the transition between C3 and lqrrt could go seamlessly and not require abunch of in file changes